WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3067,12 +3109,31 @@ public void onTrackableEvent(TrackableEvent event, Map<String, String> propertie
     }
 
     @Override public void onStoryComposerLoadRequested(ArrayList<Object> mediaFiles, String blockId) {
```

Comment:
NitPick 😅 Hi @mzorz could we put these dialogs and such forth in this method in another class? Or is it not worth it? I just glanced at the code again and it might not be worth the effort. If we could somehow simplify the dialog code with a `Util` function or something so we don't add too many lines here.

Reply:
Yeah I understand where you come from - I think the logic is simple enough to follow but it's true it's "too vertical" and exceeds the screen, to add that we don't want this class to keep growing much. Might do that at a later phase 👍 (added to my ToDo list as well)

Reply:
Just following up here - addressed this here https://github.com/wordpress-mobile/WordPress-Android/pull/13198